clang/test/Driver/riscv-arch.c

```diff
@@ -306,7 +306,7 @@
 // RUN: not %clang --target=riscv32-unknown-elf -march=rv32ixabc_ -### %s \
 // RUN: -fsyntax-only 2>&1 | FileCheck -check-prefix=RV32-XSEP %s
 // RV32-XSEP: error: invalid arch name 'rv32ixabc_',
-// RV32-XSEP: extension name missing after separator '_'
+// RV32-XSEP: unsupported non-standard user-level extension 'xabc'
```

Comment:
So there is no test coverage for `extension name missing after separator` now? Or is this error in a dead code path now?

Reply:
It's tested in llvm/unittests/TargetParser/RISCVISAInfoTest.cpp

```
TEST(ParseArchString, RejectsDoubleOrTrailingUnderscore) {                       
  EXPECT_EQ(                                                                     
      toString(RISCVISAInfo::parseArchString("rv64i__m", true).takeError()),     
      "extension name missing after separator '_'");                             
                                                                                 
  for (StringRef Input :                                                         
       {"rv32ezicsr__zifencei", "rv32i_", "rv32izicsr_", "rv64im_"}) {           
    EXPECT_EQ(toString(RISCVISAInfo::parseArchString(Input, true).takeError()),  
              "extension name missing after separator '_'");                     
  }                                                                              
}
```